opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/client/ClientTracingFeature.java

```diff
@@ -1,28 +1,45 @@
 package io.opentracing.contrib.jaxrs2.client;
 
-import java.util.Arrays;
+import io.opentracing.Tracer;
+import io.opentracing.util.GlobalTracer;
+import java.util.Collections;
 import java.util.List;
-import java.util.logging.Level;
 import java.util.logging.Logger;
-
+import javax.ws.rs.Priorities;
 import javax.ws.rs.client.Client;
-
-import io.opentracing.Tracer;
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
 
 /**
  * @author Pavol Loffay
  */
-public class ClientTracingFeature {
-
+@Provider
```

Comment:
It does not work for clients, but I found using this annotation for client's features too. (maybe a good practice to stress that it is a provider)